ahachul_backend/src/main/kotlin/backend/team/ahachul_backend/api/community/adapter/web/out/CustomCommunityPostRepository.kt

```diff
@@ -21,6 +28,73 @@ class CustomCommunityPostRepository(
     private val queryFactory: JPAQueryFactory
 ) {
 
+    fun getByCustom(postId: Long, memberId: String?): CommunityPost? {
+        return queryFactory.select(
+            Projections.constructor(
+                CommunityPost::class.java,
+                communityPostEntity.id,
+                communityPostEntity.title,
+                communityPostEntity.content,
+                communityPostEntity.categoryType,
+                ExpressionUtils.`as`(
+                    JPAExpressions.select(count(communityPostLikeEntity.id))
```

Comment:
이거 각각 메서드로 추출하면 훨씬 보기 편할 것 같아요! 아래하고 중복돼서
```suggestion 
fun getLikeOrHateCount(type: YNType) {
}
```

Reply:
넵. 이부분 추출하려고 했는데 게시물에 좋아요랑 싫어요 카운트 컬럼 추가할 것 같아서 우선 냅뒀어요.
다음 이슈에서 같이 수정할게요~